### DISCUSSION TITLE
BIG BOX in gameplay!

### DISCUSSION THREAD

**Discussion by PLAYERUNKNOWNHatsune01** [ppy/osu#16006] [Q&A] [answered]
How can I remove this BIG BOX in the center! I update osu lazer and this comes up! 
Image: https://i.ibb.co/0VrnYGG/Screenshot-from-2021-12-08-18-16-37.png

**Comment by peppy** [comment, marked as answer]
Open settings and search for "border", and disable it.

**Reply by PLAYERUNKNOWNHatsune01** [reply]
thank you sir!
